src/assets/styles/changes.sass

```diff
@@ -0,0 +1,132 @@
+.changelog
+    strong
+        color: #636363
+    
+    h3
+        color: #636363
+
+    i.fa
+        color: #585858
+        padding-right: 5px
+    
+    .fa-rss
+        padding-left: 3px
+    
+    .feed
+        height: 50px
+        margin-top: 30px
+    
+    .tagsfilter
+        max-width: 320px
+
+        h3
+            margin-top: 0px
+            font-size: 20px
+
+        .reset
+            display: block
+            margin: 10px 0
+            color: #585858
+            font-size: 12px
+    
+    .tag
+        display: inline-block
+        background-color: #999
+        border-radius: 4px
+        padding: 0px 8px
+        margin: 4px 2px
+        font-size: 14px
+        font-weight: 400
+        line-height: 26px
+
+        a
+            color: #000
+        a:hover
+            text-decoration: none
+
+    .meta
+        padding: 10px 0
+        border-top: 1px solid #ccc
+
+        .permalink
+            font-weight: 700
+            color: #585858
+            margin-right: 8px
+        
+        a.repo
+            color: #585858
+            margin-left: 8px
+        a.release
+            color: #585858
+            margin-left: 8px
+        .fa
+            font-size: 18px
+
+ul.changelog
+    list-style-type: none
+    padding-inline-start: 0
+
+    > li
+        list-style-type: none
+        margin: 30px 0
+
+        ul
+            li
+                list-style-type: disc
+
+    .content
+        font-size: 14px
+        // max-height: 200px
+        // overflow: hidden
+        // text-overflow: ellipsis
+        // content: ""
+        // position: relative
+        
+
+
+        // &:before
+        //     content: ""
+        //     width: 100%
+        //     height: 100%
+        //     position: absolute
+        //     left: 0
+        //     top: 0
+        //     background: linear-gradient(transparent 150px, white)
+        //     pointer-events: none
```

Comment:
Commented out code can be removed unless you intend to add it back before merging.

Reply:
Right. I will move it to an issue for a future front end engineer :)

Reply:
Comments are removed